snappy/snap_yaml.go

```diff
@@ -62,8 +62,8 @@ type AppYaml struct {
 }
 
 type plugYaml struct {
-	Interface           string `yaml:"interface"`
-	SecurityDefinitions `yaml:",inline"`
+	Interface string `yaml:"interface"`
+	//SecurityDefinitions `yaml:",inline"`
```

Comment:
well I really hope snap_yaml.go will go away soon, so not too worried about leaving that there